packages/babel-core/test/option-manager.js

```diff
@@ -73,8 +73,8 @@ describe("option-manager", () => {
       ).toBeTruthy();
     });
 
-    it("should throw if a plugin is repeated, with information about the repeated plugin", () => {
-      const { calls, plugin } = makePlugin("my-plugin");
```

Comment:
`makePlugin` doesnt accept any argument so this was misleading, one could think this is relevant for this test somehow

Reply:
The test above has the same problem!

Reply:
corrected this as well 👍